mods/cnc/maps/gdi06/gdi06.lua

```diff
@@ -108,15 +108,18 @@ WorldLoaded = function()
 		KillCounterHuntThreshold = 20
 	end
 
-	destroyObjective = player.AddPrimaryObjective("Destroy the Nod ********.")
+	destroyObjective = player.AddObjective("Destroy the Nod ********.", "Primary", false)
```

Comment:
Why is `required` set to false here? The mission won't end when the player fails.

Reply:
It will, because the enemies objective is completed/failed.